app/src/main/java/com/hamdan/forzenbook/viewmodels/LoginViewModel.kt

```diff
@@ -274,3 +275,13 @@ fun LoginViewModel.LoginState.toLoginEntrys(): LoginEntrys =
         email = this.email,
         code = this.code,
     )
+
+fun LoginViewModel.LoginState.toLoginComposeState(): LoginComposeState =
+    LoginComposeState(
+        email = this.email,
+        code = this.code,
+        showInfoDialog = this.showInfoDialog,
+        inputtingCode = this.inputtingCode,
+        isLoading = this.isLoading,
+        hasError = this.hasError
+    )
```

Comment:
I see all the ttf files deleted? Where did they go?

Reply:
they got moved, but they more appropriately will be in the ui Core module when merged